test/support/sample/SampleEnterpriseUnit.scala

```diff
@@ -0,0 +1,28 @@
+package support.sample
+
+import uk.gov.ons.sbr.models.enterprise.{ Enterprise, Ern }
+
+trait SampleEnterpriseUnit {
+
+  val SampleEnterpriseId = Ern("1000000012")
+  val SampleNumberOfEmployees = 100
+  val SampleJobs = 15
+  val SampleEnterpriseReference = "someEnterpriseRef"
+  val SampleEnterpriseName = "Company Name Plc"
+  val SamplePostcode = "NP0 XXX"
+  val SampleLegalStatus = "some-LegalUnit"
+
+  val DefaultDeliminator = "~"
```

Comment:
Why is this needed here?
We only want one definition of the rowKey delimiter - and that is in the HBase object.